packages/common/test-configs/setup-react.ts

```diff
@@ -1,14 +1,13 @@
+// This adds custom Jest matchers for working with the DOM
+// https://github.com/testing-library/jest-dom#custom-matchers
 import '@testing-library/jest-dom';
 
-jest.mock('react', () => {
-  const originalModule = jest.requireActual('react');
+import { AnyObject } from '../src/types/common';
```

Comment:
This is failing the Prow test, try importing just the `type` or rebase on top of https://github.com/openshift/dynamic-plugin-sdk/pull/91 (will be merged in a few so could be just enough to rebase on main)